### ISSUE TITLE
Declarative config

### ISSUE THREAD

**Issue by michaelneale** [TBD54566975/dwn-proxy-js#22]
Would be nice to have a CLI only experience where you can match up messages to HTTP/rest endpoints for incoming messages and also setup outgoing messages.

**Comment by KendallWeihe** [comment]
PR merged https://github.com/TBD54566975/dwn-proxy-js/pull/26 

A few thoughts...

- I've created a `dwn-proxy-cli.ts` file, but it's just a start... 
- I think we may be able to use the protocol definition internally, which would simplify some of the yml syntax
  - or even, perhaps some of the ideas here lead to us thinking, "maybe some of these things _ought_ to be pushed into the DWN protocol spec & SDK implementation"
- we need to use this https://github.com/TBD54566975/dwn-sql-store

**Comment by michaelneale** [comment]
@KendallWeihe nice - any rough instructions we can add to the readme for the cli? The config seems ok to me - glancing at it I was able to make sense of inbound and outbound at least

**Comment by KendallWeihe** [comment]
From my notes. Ultimately, we ought to do something like this...

```shell
$> web5 dwn
$> web5 dwn message create ...
$> web5 dwn message send ...
$> web5 dwn records ...
...

$> web dwn proxy
...

$> web5 did
...

$> web5 vc
...
```

**Comment by michaelneale** [comment]
@KendallWeihe oh nice - I didn't see that cli dir for some reason. But that is nice. 

Once the format/syntax settled down, can have a nice end to end guide for building an app (client and server) using this I think with the CLI.

**Comment by KendallWeihe** [comment]
> Once the format/syntax settled down, can have a nice end to end guide for building an app (client and server) using this I think with the CLI.

Yeah fo sho. In it's current state, it's hardly a CLI. It's a single purpose tool right now, but it'll grow 🪴
